inductiva/examples/examples.py

```diff
@@ -0,0 +1,102 @@
+"""Functions to download and load examples from URLs.
+
+It contains several samples of molecules and a test_vehicle."""
+import os
+from typing import Optional
+from urllib import request, error
+
+from absl import logging
+
+import inductiva
+
+
+def load_insulin(save_dir: Optional[str] = None) -> str:
+    """Load the insulin molecule.
+    Returns:
+        The path to the downloaded PDB file."""
+    return download_from_rcsb("1ZNI", save_dir)
+
+
+def load_hemoglobin(save_dir: Optional[str] = None) -> str:
+    """Load the hemoglobin molecule.
+    Returns:
+        The path to the downloaded PDB file."""
+    return download_from_rcsb("1A3N", save_dir)
+
+
+def load_lysozyme(save_dir: Optional[str] = None) -> str:
+    """Load lysozyme protein.
+    Returns:
+        The path to the downloaded PDB file."""
+    return download_from_rcsb("1AKI", save_dir)
+
+
+def load_test_vehicle(save_dir: Optional[str] = None) -> str:
+    """Load a test vehicle from Inductiva Github.
+
+    Returns:
+        The path to the downloaded OBJ file."""
+
+    return download_inductiva_resources("test_vehicle.obj", save_dir)
+
+
+def download_from_rcsb(pdb_id: str, save_dir: Optional[str] = None) -> str:
+    """Download a PDB file from the RCSB database according 
+    to its pdb ID.
+    Args:
+        pdb_id: The PDB identifier of the molecule to download.
+    Returns:
+        The path to the downloaded PDB file.
+    """
+    file_path = f"{pdb_id}.pdb"
+    api_url = f"https://files.rcsb.org/download/{file_path}"
+
+    file_path = download_from_external_url(api_url, file_path, save_dir)
+
+    return file_path
+
+
+def download_inductiva_resources(file_name: str,
```

Comment:
I think this file can me divided into three files. One with the fluids examples, one with the molecules examples and one with the general functions.

Reply:
OMG! We are already going for 3 files???? 
I can't take this anymore...!.... :)
What's wrong with you people?? :)